packages/compass-aggregations/src/components/focus-mode/focus-mode-stage-editor.tsx

```diff
@@ -9,9 +9,8 @@ import StageOperatorSelect from '../stage-toolbar/stage-operator-select';
 
 const containerStyles = css({
   height: '100%',
-  display: 'grid',
-  gridTemplateRows: 'min-content 1fr',
-  gridTemplateColumns: '1fr',
+  display: 'flex',
```

Comment:
If you are changing this to flex, then you should be explicit about what children you want to be flexible to avoid unpredictable behavior, in this case you probably want `headerStyles` to have flex set to `none` and `editorStyels` to have flex grow set to 1 (and you probably don't want to have height set to 100% there because this is 100% of the container, which doesn't account for the container)